include/zim/archive.h

```diff
@@ -67,6 +67,7 @@ namespace zim
        */
       explicit Archive(const std::string& fname);
 
+#ifndef _WIN32
```

Comment:
Please add a comment in the docstring.
We use the linux build to generate the doc. We must tell user that the feature is not enable on windows.

Reply:
Done